src/containers/PersonContainer/PersonRoles.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { FC } from "react";
+import { Link } from "react-router-dom";
+
+import Role from "../../models/Role";
+
+import Details from "../../components/Shared/Details";
+import H2 from "../../components/Shared/H2";
+import Ul from "../../components/Shared/Ul";
+
+import { fontSizes } from "../../styles/fonts";
+
+interface PersonRolesProps {
+  /** The person's councilmember roles */
+  councilMemberRoles: Role[];
+  /** The person's non councilmember roles, partitioned by concilmember roles' date range
+   * and whether the role is active or inactive.
+   * The first list of roles are active, while the second list of roles are inactive.
+   */
+  nonCouncilMemberRoles: Record<string, Role[][]>;
+}
+
+const PersonRoles: FC<PersonRolesProps> = ({
```

Comment:
Top level comment: curious as to what happens when no roles are stored for a person. What gets displayed?

Reply:
The top level page catches the error -- displays `Person is not a council member` See "Daniel strauss" page

Reply:
Our data model allows for people to be added with no Role + Seat info though. We can still track voting information for a person even if they dont have any role stored. So we definitely dont want to error.

Reply:
Ok, here's Daniel Strauss page (he has roles nor matter sponsored)

![image](https://user-images.githubusercontent.com/37560480/153101025-0007dab5-2646-4102-b268-ca1d22ad2bba.png)

Reply:
No contact info either

Reply:
That honestly works for me :joy:

Reply:
Thanks for fixing!